### ISSUE TITLE
[FEAT] - Audit Log forwarding for central cluster audit logs

### ISSUE THREAD

**Issue by IvoGoman** [cloudoperators/greenhouse#415]
### Priority

(Medium) I'm annoyed but I'll live

### Description

It is necessary to forward the audit logs of our central cluster to the internal logsink we are using for the other clusters.
This is required to drive the RBAC adoption.

We need to investigate how to ship logs from the gardener api server to octobus/elastic search.

**Comment by uwe-mayer** [comment]
It seems Gardener only supports audit log forwarding to `SAP BTP Audit Log Service` atm:
https://github.tools.sap/kubernetes/gardener-extension-shoot-auditlog-service/issues/135

To unblock us from proceeding with https://github.com/cloudoperators/greenhouse/issues/388
we will forward all [TeamRoleBindingController](https://github.com/cloudoperators/greenhouse/blob/main/pkg/controllers/teamrbac/teamrolebinding_controller.go) logs to Octobus and audit log priviledge escalation there.

**Comment by IvoGoman** [comment]
Closing in favor of #485
